wiki/Reporting_Bad_Behaviour/Handling_Cheating/en.md

```diff
@@ -1,74 +1,73 @@
-Why am I banned?
-----------------
+# Handling Cheating
 
--   10 silences issued in your userpage.
--   You gave your account to somebody else.
+
+## Why am I restricted?
+A user may get restricted if they do any of the following:
+-   Continuous misbehaviour regardless of how many infringements they accumulate.
+-   Circumventing a punishment by making a new account.
+-   Sharing their account with other users.
 -   Using multiple accounts.
--   Cheating (see below).
+-   Hacking or cheating (see below).
 
-What is considered cheating?
-----------------------------
 
-As the rules says, any form of manipulation of the client; score hacking, macro-enabled full-combo sequences, and auto-aim are all considered cheating.
+## What is considered cheating?
+The [rules](/wiki/Rules) say that any form of manipulation of the client such as score hacking, macro-enabled full-combo sequences, auto-aim and anything else that plays the game for the player.
 
-This is a one-strike rule; cheaters will be **permanently banned.**
+This is a one-strike rule. Cheaters will be restricted on their first offence for an indefinite amount of time. The team may decide to lift their restriction if they successfully appeal.
 
-I have been banned for cheating. I didn't cheat!
-------------------------------------------------
 
-osu! uses an automated metrics combined with manual confirmation.
+## I have been restricted for cheating. I didn't cheat!
+osu! uses automated metrics combined with manual confirmation. If you don't know why you are restricted, contact [accounts@ppy.sh](mailto:accounts@ppy.sh).
 
-The staff never places an incorrect ban for cheating. Before they do any actions, they make sure it first 100%.
 
-My friend has been banned for cheating! They definitely wouldn't cheat! You are wrong!
---------------------------------------------------------------------------------------
+## Prove that I/they were cheating! Tell us how you know!
+The team will not provide any proof. Giving away the countermeasures would make them useless in further detecting cheaters.
 
-It may be hard to come to terms with, but you will have to trust the staff. peppy is not trying to tear apart friendships but he knows that nearly no one can be trusted. Even those people who are close to you that you may never suspect can be just acting behind you or any other person.
 
-Prove that I/they were cheating! Tell us how you know!
-------------------------------------------------------
+## I'll make a video to prove my innocence!
+Sorry but if you have been restricted, they are already 100% sure you are cheating. A video will not help because even these can be also doctored.
```

Comment:
"they are already 100% sure you are cheating" ???? **who** is sure;
comma before `but` and `because`

Reply:
`Sorry, but if you have been restricted, the team is already sure you are cheating.` ?

Reply:
yeah